src/core/components/SlideInBar/SlideInBar.tsx

```diff
@@ -32,17 +34,15 @@ export const SlideInBar = ({ children, open, title, handleClose, from }: SlideIn
     { [`slide-in--hidden-${direction}`]: !open }
   );
 
-  const overlayClasses = classnames('slide-in-menu-overlay', { [`slide-in-menu-overlay--${'visible'}`]: open });
+  const overlayClasses = classnames('slide-in-menu__overlay', { [`slide-in-menu__overlay--${'visible'}`]: open });
 
   return (
     <div className="root slide-in-menu">
       <div role="button" tabIndex={-1} onClick={handleClose} onKeyDown={handleClose} className={overlayClasses} />
       <div className="slide-in-menu">
         <div className={slideInBarClasses}>
-          <div className="slide-in-nav-header">
-            <span className="slide-in-title">{title}</span>
-          </div>
-          <div className="slide-in-menu-item-list-block">{children}</div>
+          <SlideInBarMenuHeader title={title} componentName={componentName} />
```

Comment:
We use the `componentName` to define the visibility of some inner component of the SlideInBarMenuHeader. Lets instead define the terms inside of `SlideInBar ,` and pass the result of the check as a boolean value to the `SlideInBarMenuHeader `.

Reply:
Component updated!